app/assets/stylesheets/pages/registration.scss

```diff
@@ -0,0 +1,12 @@
+//popover css used in signup page for client side validation
```

Comment:
Github hided my comment regarding this.

"That's what I thought. So, these styling can be removed then.

I didn't see any form validation using popover. It's using the built-in validators. I think you can use setCustomValidity method to achieve the same look and feel.

What do you think, @mssola?"